instances/devhub.near/widget/devhub/entity/proposal/Editor.jsx

```diff
@@ -439,9 +439,23 @@ useEffect(() => {
         Array.isArray(proposalIdsArray) &&
         proposalIds.length !== proposalIdsArray.length
       ) {
-        setCreateTxn(false);
-        setProposalId(proposalIds[proposalIds.length - 1]);
-        setShowProposalPage(true);
+        const latestProposalId = proposalIds[proposalIds.length - 1];
+        const latestProposal = Near.view(
+          "${REPL_DEVHUB_CONTRACT}",
+          "get_proposal",
+          {
+            proposal_id: latestProposalId,
+          }
+        );
+
+        if (
+          latestProposal.snapshot.name === title &&
+          latestProposal.snapshot.description === description
+        ) {
+          setCreateTxn(false);
```

Comment:
can you please make this change for `events` and `infra` proposal and RFP, since there also we'll have the same issue.

Reply:
Thanks for pointing out. I have made a separate issue for `events` and `infra` here: https://github.com/NEAR-DevHub/neardevhub-bos/issues/944

Also pointing out that we need to make sure that tests for creating and submitting a proposal have to be common for all 3 instances.